### ISSUE TITLE
Internal Error when trying to install native-image in Graalvm ee

### ISSUE THREAD

**Issue by olepoeschl** [oracle/graal#3144]
I have the GraalVM Enterprise Edidtion installed.
When I type "gu install native-image", it produces following Error-Message:

Internal error occurred: Illegal char <:> at index 5: https:\oca.opensource.oracle.com\gds\native-image\license.txt

For me it looks like there's just missing a backslash.

**Steps to reproduce the issue**
1. Download Graalvm ee
2. type "gu install native-image"

**Describe GraalVM and your environment:**
 - GraalVM version 21.0.0
 - JDK major version: Java11
 - OS: Windows10
 - Architecture: AMD64

```
        C:\Program Files\Java\graalvm-ee-java11-21.0.0\bin>gu install native-image
Downloading: Release index file from oca.opensource.oracle.com
Downloading: Component catalog for GraalVM Enterprise Edition 21.0.0 on jdk11 from oca.opensource.oracle.com
Skipping ULN EE channels, no username provided.
Downloading: Component catalog from www.graalvm.org
Processing Component: Native Image
Internal error occurred: Illegal char <:> at index 5: https:\oca.opensource.oracle.com\gds\native-image\license.txt
```

**Comment by olepoeschl** [comment]
my fault

**Comment by sdedic** [comment]
@olepoeschl could you please run the command with `-e` switch and provide the complete error stacktrace ? Will help to trace/fix all occurrences of the issue.

**Comment by olepoeschl** [comment]
It says

`Internal error occurred: Illegal char <:> at index 5: https:\oca.opensource.oracle.com\gds\native-image\license.txt
java.nio.file.InvalidPathException: Illegal char <:> at index 5: https:\oca.opensource.oracle.com\gds\native-image\license.txt
        at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
        at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:229)
        at java.nio.file.Path.of(Path.java:147)
        at java.nio.file.Paths.get(Paths.java:69)
        at org.graalvm.component.installer.SystemUtils.fromArray(SystemUtils.java:226)
        at org.graalvm.component.installer.SystemUtils.fromCommonRelative(SystemUtils.java:215)
        at org.graalvm.component.installer.commands.InstallCommand.addLicenseToAccept(InstallCommand.java:256)
        at org.graalvm.component.installer.commands.InstallCommand.processComponents(InstallCommand.java:498)
        at org.graalvm.component.installer.commands.InstallCommand.prepareInstallation(InstallCommand.java:526)
        at org.graalvm.component.installer.commands.InstallCommand.executeStep(InstallCommand.java:564)
        at org.graalvm.component.installer.commands.InstallCommand.execute(InstallCommand.java:194)
        at org.graalvm.component.installer.ComponentInstaller.doProcessCommand(ComponentInstaller.java:431)
        at org.graalvm.component.installer.ComponentInstaller.processCommand(ComponentInstaller.java:445)
        at org.graalvm.component.installer.ComponentInstaller.run(ComponentInstaller.java:569)
        at org.graalvm.component.installer.ComponentInstaller.main(ComponentInstaller.java:622)`

**Comment by mcraj017** [comment]
@olepoeschl PLease download the native image from below link 
https://www.oracle.com/downloads/graalvm-downloads.html?selected_tab=21
and install using
gu install -L <path\native-image jar file>

**Comment by olepoeschl** [comment]
Thank you!

**Comment by sdedic** [comment]
Colleagues, would you please kindly leave this issue open, since the behavious **is a bug** which I am going fix - so this ticket should serve to tracking delivery of the actual fix ? Thank you for understanding.

**Comment by sdedic** [comment]
Although reported against GraalVM EE, the culprit code is actually part of the Github sources; so should be fixed here.

**Comment by sdedic** [comment]
This should be fixed in `21.0.0.2` release.
